permutableString.js

```diff
@@ -0,0 +1,34 @@
+// https://dmitripavlutin.com/how-to-compare-objects-in-javascript/
+//Link above - compare objects, 3 subheader - don't clear 'Object.keys'...
+//Works when both of strings are the same length.
+function permutable(s1, s2) {
+    let obj1 = {}, obj2 = {};
+
+    function countSymbols(str, obj) {
```

Comment:
I think you don't need to pass `obj` in an argument.
Please refactor to `countSymbols(str)` so it can be used as
obj1 = countSymbols(str)